_posts/2024-01-06-developer-mission-statement.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+id: 4c439aca-9f8f-4e66-9589-293f0481ab1a
+title: My Mission Statement as a Programmer
+author: Mick F
+excerpt: >-
+  Climate Friendliness, Human Friendliness, Exigence, IndieWeb Friendliness, Big
+  Corporation Cautiousness, Durability & Sustainability serves as a guiding
+  compass in my journey as a programmer
+category: Column
+tags:
+  - iOS
+  - Swift
+---
+
+## tl;dr
+
+As a programmer, I create eco-friendly software with user-friendly design,
+embracing IndieWeb principles. I'm cautious with big corporations, upholding
+ethics and user rights. My goal is durable, sustainable code, prioritizing
+maintainability and developer well-being. I respond swiftly to evolving
+challenges for impactful work in a changing tech landscape.
+
+## A longer version
+
+Let me explain. There is a sense of uneasiness with the state of the world these
+days. And after I had the privilege to spend a fun and peaceful holiday season,
+I also wanted to give a headcheck on where I stand with my role as a programmer
```

Comment:
```suggestion
I wanted to do a headcheck on where I stand with my role as a programmer
```

Reply:
Not sure if "headcheck" is the right word, but we get it and I can't think of a better term